components/tikv_util/src/logger/mod.rs

```diff
@@ -31,13 +32,38 @@ pub fn init_log<D>(
     level: Level,
     use_async: bool,
     init_stdlog: bool,
+    mut disabled_targets: Vec<String>,
 ) -> Result<(), SetLoggerError>
 where
     D: Drain + Send + 'static,
     <D as Drain>::Err: std::fmt::Display,
 {
+    // Only for debug purpose, so use environment instead of configuration file.
+    if let Ok(extra_modules) = env::var("TIKV_DISABLE_LOG_TARGETS") {
+        disabled_targets.extend(extra_modules.split(',').map(ToOwned::to_owned));
+    }
+
+    let filtered = drain.filter(move |record| {
+        if disabled_targets.is_empty() {
```

Comment:
Should check !empty?

Reply:
my fault, make it wrong when addressing comments